shuffle/cli/src/forge_test_helper.rs

```diff
@@ -1,35 +1,41 @@
 // Copyright (c) The Diem Core Contributors
```

Comment:
Do we want test code in actual src. Has this been done somewhere else in the repo?

shuffle/cli/src/forge_test_helper.rs

Reply:
We do have test_helpers in some src folders. Some or files, a folder, or it's own crate

<img width="554" alt="CleanShot 2021-11-22 at 08 52 47@2x" src="https://user-images.githubusercontent.com/635121/142902373-2e14bd20-fd43-4566-a173-8f197251df29.png">